File: Python Programming/problems on numbericals/problem11.py
'''
11.Write a program which accept number from user and display its table. 
	
	Author : Milind Nalawade
	
	Date : 24/6/2021
	
	language : Python 3.9.0
	
'''	

#======
#Logic
#======

def Table(no):
	j=1
	ans=1
	while j<11:
		ans=no*j
		print(ans,end=" ")
		ans=1
		j=j+1
#=============
#Entry point function
#=============
def main():
	ino=int(input("Enter the number : "))
	Table(ino)
#=======
#starter
#=======
if __name__=="__main__":
	main()

File: C Programming/problem on numbericals/problem11.c
/*
	11.Write a program which accept number from user and display its table. 
	
	Author : Milind Nalawade
	
	Date : 24/6/2021
	
	language :C programming
	
*/

#include<stdio.h>

//logic

void Tables(int no)
{
	int j=1;
	int ans=1;
	while(j<11)
	{
		ans=no*j;
		printf("%d ",ans);
		ans=1;
		j++;
	}	
}

//Entry point function

int main()
{
	int i=0;
	printf("Enter the number : ");
	scanf("%d",&i);
	Tables(i);
	return 0; 
}